config/routes.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  get 'greetings/index'
+  root 'root#index'
+end
```

Comment:
- [ ] You might need to look into this route too. This route only generates this message alone, not random messages I created.

### For `http://127.0.0.1:3000/greetings/index`
![msg-1](https://github.com/akezeth/hello-rails-react/assets/26597432/fc2b7e94-b803-465e-a6c9-c47c1099d32b)

### For `http://127.0.0.1:3000/greetings`

![msg-2](https://github.com/akezeth/hello-rails-react/assets/26597432/26f393c4-88cc-4de5-8bdd-7b1bed54979b)

### For `http://127.0.0.1:3000`

![loading](https://github.com/akezeth/hello-rails-react/assets/26597432/680fa033-5cb3-4aa3-aa79-87c9296fc729)